API.md

```diff
@@ -50,6 +46,9 @@ attached to `getBlock` are errored with `Error('manual unwant: key')`.
 
 Cancel previously requested keys.
 
+### `putStream()`
+
+Returns a duplex `pull-stream` that emits an object `{key: Multihash}` for every written block when it was stored.
 
 ### `hasBlock(block, cb)`
```

Comment:
same goes for `s/hasBlock/has`. However, should bitswap have this call? What would be the case?